src/user/user.service.ts

```diff
@@ -121,15 +121,16 @@ export class UserService {
     return res;
   }
 
-  async uploadProfilePicture(uid: number, image: Express.Multer.File): Promise<UploadProfileResponse> {
+  async uploadProfilePicture(uid: number, image: UploadProfileRequest): Promise<UploadProfileResponse> {
     const { uid: foundUid } = await this.findOne({ uid: uid }, { shouldExist: true });
-    if (!image || !ALLOWED_PROFILE_FORMAT.includes(image.mimetype)) {
+    if (!image || !ALLOWED_PROFILE_FORMAT.includes(image.type)) {
       throw new UnsupportedMediaTypeException('Invalid image type');
     }
-    if (image.size > 20000000) {
+    const buffer = Buffer.from(image.base64);
+    if (buffer.byteLength > 20000000) {
```

Comment:
check with point again. I know from him that I should be 5 MB

Reply:
Let's do 5,000,000 for 5MB?

Reply:
Sure